daemon/execdriver/native/driver.go

```diff
@@ -51,6 +52,12 @@ func NewDriver(root, initPath string, options []string) (*driver, error) {
 		return nil, err
 	}
 
+	if apparmor.IsEnabled() {
+		if err := hasAppArmorProfileLoaded("docker-default"); err != nil {
+			return nil, fmt.Errorf("AppArmor enabled on system but the docker-default profile is not loaded. Install policies from contrib/apparmor (https://goo.gl/CjGtgC) and try again.")
```

Comment:
@ewindisch one of us forgot the is in "AppArmor enabled"  which should be "AppArmor is enabled"   --- minor nit. If you don't want to kill it.  I  understand. Otherwise LGTM